manifests/filter/ip_user.pp

```diff
@@ -0,0 +1,102 @@
+# == Resource: repose::filter::ip_user
+#
+# This is a resource for generating ip identity configuration files
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool.  Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String.  Config filename
+# Defaults to <tt>ip-user.cfg.xml</tt>
+#
+# [*quality*]
+# List of filters by name, which conains a list of headers
```

Comment:
Addressed in amended commit, copy pasta error, it is also incorrect at the source (the filter this replaces in repose 8) the `ip_identity` filter docs will need to be corrected at some point too.